test/lib/mayaUsd/render/mayaToHydra/testMayaDisplayModes.py

```diff
@@ -48,42 +48,51 @@ def test_MayaDisplayModes(self):
                 "testMayaDisplayModes",
                 "testMayaDisplayModes.ma")
         cmds.refresh()
-
         panel = mayaUtils.activeModelPanel()
         
+        
         cmds.modelEditor(panel, edit=True, wireframeOnShaded=False)
 
         #Use Default Material
         cmds.modelEditor(panel, edit=True, useDefaultMaterial=True)
         cmds.refresh()
         self.assertSnapshotClose("defaultMaterial" + ".png", self.imageDiffFailThreshold, self.imageDiffFailPercent)
+        cmds.refresh()
+
+        #xray mode
+        cmds.modelEditor(panel, edit=True, useDefaultMaterial=False)
+        cmds.modelEditor(panel, edit=True, xray=True)
+        cmds.refresh()
+        self.assertSnapshotClose("xray" + ".png", self.imageDiffFailThreshold, self.imageDiffFailPercent)        
+        cmds.modelEditor(panel, edit=True, xray=False)
 
+        #Use Default Material
+        cmds.modelEditor(panel, edit=True, useDefaultMaterial=True)
```

Comment:
Here you are setting the default material on but never remove it, is that on purpose ? I think we should have at line 71 :
 cmds.modelEditor(panel, edit=True, useDefaultMaterial=False)

Reply:
yes, made some changes wrt to defaultMaterial usage. also helps make the images look distint.